.github/workflows/ci-badger-bank-tests-nightly.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+      - release/v3.2103
   schedule:
```

Comment:
@skrdgraph  Is globbing possible here?  i.e. `- release/* `?

Reply:
Lets do this next time, because I delayed you here!